src/graphics.c

```diff
@@ -8,6 +8,7 @@ See the file LICENSE for details.
 #include "kerneltypes.h"
 #include "font.h"
 #include "kernelcore.h"
+#include "math.h"
```

Comment:
You should wait for my math pr to get merged, then add it on top. We can't merge this in yet because it doesn't compile (right?). The point of the master branch is that it should always build and be ready for release.